lib/web_ui/test/html/shaders/linear_gradient_golden_test.dart

```diff
@@ -86,7 +86,7 @@ Future<void> testMain() async {
     }
     expect(rc.renderStrategy.hasArbitraryPaint, isTrue);
     await canvasScreenshot(rc, 'linear_gradient_oval_matrix');
-  });
+  }, skip: isFirefox);
```

Comment:
Did this suddenly start failing on Firefox? Thoughts @yjbanov ?

Reply:
I'm surprised this wasn't failing before. We don't have infra to golden-test Firefox. @eyebrowsoffire reorged our tests recently and might know more.

Reply:
I don't think these tests will fail on Firefox, but they will not take screenshots or send them to skia gold. However, it may be of at least some utility to just run the tests on Firefox anyway just to make sure you can actually do the drawing commands and not crash or throw an exception. I'll leave it up to you though, it might not be worthwhile to run them.